### PR TITLE
Ready: Use new timeseries float/double apis

### PR DESCRIPTION
**Pull request by alexmoore**
Rework client to use new float/double api for cells. Swap "Numeric" column type for "Float".

### CODE REVIEW

**Comment by alexmoore** [comment]
Only question on this is whether we should allow numeric strings in the case when a column description is defined. 

**Example**
For a row that has 1 cell, SQL type Float.   
1. Create Storevalue command
2. withColumns = [TsColumnType.Float]
3. withRows = [ [ '4.5' ] ]  

Storevalue command would then put '4.5' in the numericValue register of the cell only in this case.  
Without the columns definition, it would store the string value '4.5' to the string cell register.

See https://github.com/basho/riak-nodejs-client/pull/105/files#diff-f26b0b0dee3b0c66618f90c8dfddf3bbR147.

File: lib/commands/ts/data.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var validColumnTypes = Object.freeze([0, 1, 2, 3, 4, 5, 6]);
 var ColumnType = Object.freeze({
     Binary:    TsColumnType.BINARY,
     Integer:   TsColumnType.INTEGER,
-    Numeric:   TsColumnType.NUMERIC,
+    Float:   TsColumnType.FLOAT,
```
Comment (lukebakken) [inline lib/commands/ts/data.js]: Can haz column alignment?

File: lib/commands/ts/storevalue.js [hunk 2]
```diff
@@ -144,8 +144,12 @@ StoreValue.prototype.constructPbRequest = function() {
                             // NB: must be convertible to Long to serialize as sint64
                             tsc.setIntegerValue(cell);
                             break;
-                        case TsColumnType.NUMERIC:
-                            tsc.setNumericValue(new Buffer(cell.toString()));
+                        case TsColumnType.FLOAT:
+                            if(utils.isString(cell)) {
```
Comment (lukebakken) [inline lib/commands/ts/storevalue.js]: OMG space after the `if`

File: lib/commands/ts/storevalue.js [hunk 3]
```diff
@@ -144,8 +144,12 @@ StoreValue.prototype.constructPbRequest = function() {
...
+                                tsc.setNumericValue(new Buffer(cell.toString()));
```
Comment (lukebakken) [inline lib/commands/ts/storevalue.js]: I really don't like using the numeric field and relying on the fact that Riak stuffs this data into a float. I would rather do the correct conversion client-side. What do you think?

File: test/unit/ts/data.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ var Long = require('long');
 var columns = [
     { name: 'col_binary',    type: TS.ColumnType.Binary },
     { name: 'col_int',       type: TS.ColumnType.Integer },
-    { name: 'col_numeric',   type: TS.ColumnType.Numeric },
+    { name: 'col_float',   type: TS.ColumnType.Float },
```
Comment (lukebakken) [inline test/unit/ts/data.js]: I feel like `TS.ColumnType.Numeric` should be nuked.

Comment (alexmoore) [inline test/unit/ts/data.js]: It has been :)

File: lib/commands/ts/storevalue.js (hunk 3 above)
Comment (alexmoore) [inline lib/commands/ts/storevalue.js]: Yeah, it felt like a hack at best.  I'll take it out for now.

File: lib/commands/ts/data.js (hunk 1 above)
Comment (alexmoore) [inline lib/commands/ts/data.js]: It WAS aligned. What did you do Sublime Text?  BAD Sublime Text!

File: test/unit/ts/data.js (hunk 4 above)
Comment (lukebakken) [inline test/unit/ts/data.js]: Oh lol I can read a diff, really. I was distracted by the haphazard column alignment :-P

File: lib/commands/ts/data.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ var validColumnTypes = Object.freeze([0, 1, 2, 3, 4, 5, 6]);
 var ColumnType = Object.freeze({
     Binary:    TsColumnType.BINARY,
     Integer:   TsColumnType.INTEGER,
-    Numeric:   TsColumnType.NUMERIC,
+    Float:     TsColumnType.FLOAT,
```
Comment (lukebakken) [inline lib/commands/ts/data.js]: (yey)

**Comment by lukebakken** [comment]
:ship: :it:

**Pull request merged**